stream_alert_cli/logger.py

```diff
@@ -29,13 +30,17 @@ class SuppressNoise(logging.Filter):
     """
 
     def filter(self, record):
-        suppress_starts_with = (
-            'Starting download from S3',
-            'Completed download in'
+        suppressed_messages = (
+            'Starting download from S3*',
+            'Completed download in*',
+            '*triggered an alert on log type*',
+            'Successfully sent * messages to Firehose*'
         )
+
+        message = record.getMessage()
+
         return (
-            not record.getMessage().startswith(suppress_starts_with) and
-            'triggered an alert on log type' not in record.getMessage()
+            not any(fnmatch(message, suppression) for suppression in suppressed_messages)
```

Comment:
why not use `in_set`? 😉

Reply:
I'd rather not import a helper designed for rules into the CLI

Reply:
but why?

Reply:
Because if we ever do decide to start trying to use pip, we're going to have a real hard time packaging all of this if there are weird dependencies like that. and also, it's one line

Reply:
![giphy-7](https://user-images.githubusercontent.com/11466941/31094197-2d78c6b2-a769-11e7-8175-ffdaa1ba7518.gif)

Reply:
lamo